NAMESPACE

```diff
@@ -1,6 +1,5 @@
 # Generated by roxygen2: do not edit by hand
 
-export(checkRecodeList)
```

Comment:
I have a slight preference for exporting the validation function, for the following reasons:
- there is no harm in exporting it
- the function documents our definition/technical specification of `recodeLists` (further documentation can be added in the respective help page)
- it can be used in other packages without ugly work arounds (this is currently a bit annoying in `eatFDZ`, as `eatGADS` does not export its validation functions)

Reply:
Ok sure, i was thinking of it as an internal function, but that makes sense.